### ISSUE TITLE
Pinot S3 Deep Storage - High Availability for Controllers

### ISSUE THREAD

**Issue by FranMorilloAWS** [apache/pinot#11373]
If i configured S3 to be used for Deep Storage and have a 3 node Zookeeper Cluster, if i want to also have the Controller in High Availability, is that enough, or do I require to use EFS or an HDFS cluster. Wondering in how an actual Production Environment would look like on AWS

**Comment by Jackie-Jiang** [comment]
In production environment, we'd suggest having 3 or more controllers to guarantee high availability. S3 should be good as the deep storage.

**Comment by FranMorilloAWS** [comment]
Yes, but in the documentation it says that all controllers should be using a shared file system, such as HDFS etc. Wondering if having the 3 controllers assigned to the same cluster with zookeeper and all of them using S3 as deep storage will still give me that same high availability?

**Comment by Jackie-Jiang** [comment]
Yes, as long as S3 has high availability. Essentially we want a shared file system with equal or higher availability then your SLA
